tests/test_freshness_date_parser.py

```diff
@@ -97,7 +97,6 @@ def test_relative_past_dates_with_time_as_period(self, date_string, ago, period)
         param('nine hours ago', ago={'hours': 9}, period='day'),
         param('three week ago', ago={'weeks': 3}, period='week'),
         param('eight months ago', ago={'months': 8}, period='month'),
-        param('1mon ago', ago={'months': 1}, period='month'),
```

Comment:
Similar to how there is `test_insane_dates` and `test_dates_not_supported_by_date_time`, we could add a `test_known_issues` that keeps this test case around but expects it to fail, and include a comment pointing to the issue that caused us to break it.

(I thought we were using pytest’s parametrize when I suggested marking this line as an expected failure)